crates/volta-core/src/tool/node/resolve.rs

```diff
@@ -42,10 +42,14 @@ cfg_if! {
 pub fn resolve(matching: VersionSpec, session: &mut Session) -> Fallible<Version> {
     let hooks = session.hooks()?.node();
     match matching {
-        VersionSpec::Latest => resolve_latest(hooks),
-        VersionSpec::Lts => resolve_lts(hooks),
         VersionSpec::Semver(requirement) => resolve_semver(requirement, hooks),
         VersionSpec::Exact(version) => Ok(version),
+        VersionSpec::None | VersionSpec::Tag(VersionTag::Lts) => resolve_lts(hooks),
+        VersionSpec::Tag(VersionTag::Latest) => resolve_latest(hooks),
+        VersionSpec::Tag(VersionTag::LtsRequirement(req)) => resolve_lts_semver(req, hooks),
+        VersionSpec::Tag(VersionTag::Custom(tag)) => {
+            Err(ErrorDetails::NodeVersionNotFound { matching: tag }.into())
+        }
```

Comment:
Can we add a comment here? It wasn't initially clear to me why this was always an error—I only figured it out when I realized this was Node-specific logic, and that Node doesn't have tagged releases in the same way as Node packages do.

Reply:
Absolutely